windows/src/test/unit-tests/kmcomp/test_namedcodeconstants.kmn

```diff
@@ -0,0 +1,23 @@
+c This tests #4423 -- single character stores that trigger the named character constants
+c code and resulted in a codepage conversion error.
+
+store(&VERSION) '10.0'
+
+begin Unicode > use(main)
+
+group(main) using keys
+
+store(&includecodes) 'test_namedcodeconstants.txt'
+
+store(főő) '0'
+store(bar) '1'
+store(smp) U+12345
+
```

Comment:
Would a storename of SMP characters also be a valid name to test?

Reply:
🤣 yes I guess it would. Crazy man!